#진료 목록 UI
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>330</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>새굴림</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>희망 진료</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_1">
     <item>
      <widget class="QCheckBox" name="checkBox_0">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>고혈압</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_1">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>당뇨</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>빈혈</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>위염</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>식도염</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>장염</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>변비</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>염좌</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>골절</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>탈구</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>타박상</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_11">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>관절염</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>101</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_12">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>감기</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_17">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>중이염</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_16">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>알레르기</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_15">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>아토피</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_13">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>비염</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_14">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>천식</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_18">
       <property name="font">
        <font>
         <family>새굴림</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>기타</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="choice_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
